app/controllers/ChatroomController.scala

```diff
@@ -0,0 +1,19 @@
+package controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.mvc.{AbstractController, ControllerComponents}
+
+@Singleton
+class ChatroomController @Inject()(cc: ControllerComponents)
+  extends AbstractController(cc) {
+  def create = Action(parse.json) { request =>
+    val option = (request.body \ "name").asOpt[String].map { name =>
```

Comment:
No need to map
```suggestion
    val name = (request.body \ "name").asOpt[String].get
```
then use `name` to `createRoom`